packages/shopify-codemod/test/fixtures/conditional-assign-to-if-statement/assign-to-other-identifier.input.js

```diff
@@ -0,0 +1,7 @@
+this.foo || (this.bar = 'lol');
```

Comment:
Not sure if out of scope, but will this work with multiple conditions? `this.foo || this.bar || (this.baz = 'qux')`